src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -113,6 +115,32 @@ function isPreview() {
     return $(document).find('.navbar').text().indexOf('Preview') !== -1;
 }
 
+function isModeration() {
+    return $('#moderationHintButton').length !== 0;
+}
+
+function bindModerationHintButton() {
+    if (!isModeration()) {
```

Comment:
@JosephineKwa That would result the whole logic wrapping inside the if statement. I am trying to use guard statement to avoid this (to separate logic and error handling in some sense).